src/scripts/main.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const div = document.createElement('div');
+
+  div.className = `notification ` + type;
```

Comment:
```suggestion
  div.className = `notification ${type}`;
```
or 

```suggestion
  div.className = 'notification ' + type;
```

I'd prefer the first case